src/main/scala/com/advancedtelematic/director/http/DeviceResource.scala

```diff
@@ -79,8 +73,11 @@ class DeviceResource(extractNamespace: Directive1[Namespace],
         }
       } ~
       get {
+        path(IntNumber ~ ".root.json") { version =>
+          logDevice(ns, device) { fetchRoot(ns, version) }
```

Comment:
Don't need to `logDevice` so we don't publish too many `DeviceSeen`.

Reply:
Added that because we are also doing that below for the unversioned root.json. I'll remove the logging there as well, then.

Reply:
It should be there below. So `DeviceSeen` when device gets `root.json` (which is what it must check every time) but not any other metadata like `timestamp`/`target`/`1.root`.

Reply:
Ah ok.